_posts/2019-10-05-amit.md

```diff
@@ -0,0 +1,28 @@
+---
+name: "Amitvikram Dwivedi"
+github_id: "batflarrow"
+---
+
+Heya everyone, myself batflarrorw, the combination of Batman Flash and Arrow.After two years of hell in jee coaching, i finally reached here.I thought I'll live "Majegi ki Zindagi", but frankly its the same hell:(.We werent allowed to use mobiles and basically internet before, but now even though we have devices the internet is pretty much shit.Sala har mahine ek exam daaldete hain, kya pure saal ham padhte hi rhe kya. These kind of workshops and fest are the only thing that separates us from the shithole of a life we are living.I would really like to thank our seniors for organising such events and bringing us away from hell.
```

Comment:
I really appreciate the comment 😄  But make sure it is only 1-2 lines, 10-20 words. And also remove the other lines which are not needed.